src/assets/styles/global.scss

```diff
@@ -20,6 +20,17 @@
   --calcite-ui-opacity-disabled: 0.5;
 
   --calcite-panel-width-multiplier: 1;
+  --calcite-alert-dismiss-progress-background: #{rgba($blk-000, 0.8)};
+  --calcite-button-transparent-hover: rgba(0, 0, 0, 0.05);
+  --calcite-button-transparent-press: rgba(0, 0, 0, 0.08);
+  --calcite-chip-solid-bg-light: #{rgba($ui-text-inverse-light, 0.1)};
+  --calcite-chip-solid-bg-dark: #{rgba($ui-text-inverse-dark, 0.1)};
+  --calcite-chip-font-color: #{$ui-text-1-light};
+  --calcite-chip-transparent-hover: rgba(0, 0, 0, 0.05);
+  --calcite-chip-transparent-press: rgba(0, 0, 0, 0.08);
+  --calcite-input-message-floating-background: #{rgba($ui-foreground-1-light, 0.96)};
+  --calcite-link-blue-underline: #{rgba($h-bb-070, 0.4)};
+  --calcite-scrim-background: #{rgba($blk-000, 0.75)};
```

Comment:
chip and button use the same transparent colors here, should we combine?

Reply:
chips should not have any transparency

Reply:
yep my bad